input/docs/extending/addins/best-practices.md

```diff
@@ -169,36 +169,42 @@ Until early 2019, the recommendation was to reference the Cake Contrib icon host
 This recommendation changed once again in the fall of 2019 when NuGet started supporting embedded icons.
 :::
 
+## NuGet compatible tags
+
+**_§4.1_** **Do** add `cake-addin` to the NuGet-Tags.
```

Comment:
Continue with 3.x (as it is still part of the package metadata section)

Reply:
👍